### DISCUSSION TITLE
I didn't get the benefits after being approved since 5 days

### DISCUSSION THREAD

**Discussion by hemant2807** [community/community#146036] [GitHub Education]
### Hi everyone,

Hi everyone,
I still have NOT obtained the benefit of education copilot after verification approved. Does anyone have met up with this issue or who should I ask? Thank you very much.

**Comment by moghoneim-code** [comment]
Processing Time: After approval, it typically takes up to 72 hours for educational benefits, including access to GitHub Copilot, to become active. However, during peak times, such as the start of a new semester, this process may take longer.

source : https://github.com/orgs/community/discussions/55559

**Reply by hemant2807** [reply]
the wait is already more than 72 hours

**Comment by davevad93** [comment]
Hi @hemant2807 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @hemant2807 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
